src/components/controls/Icon.tsx

```diff
@@ -122,7 +122,7 @@ export const SpinnerIcon = ({...props}: IconProps) => (
 );
 
 export const DeleteIcon = ({...props}: IconProps) => (
-  <Icon fa="fa fa-minus-circle" color="#CC1100" title={t('delete')} {...props} />
+  <Icon fa="fa fa-trash" size={props.size || 2} color="#CC1100" title={t('delete')} {...props} />
```

Comment:
Dit heeft ook niets te maken met de story die je nu hebt...

Met zo'n wijzigingen moet je ook opletten, als je zo'n dingen begint te doen op het project kan dit in je gezicht ontploffen.
--> Als je denkt dat icon X beter is als icon Y: maak een PR met enkel die wijziging waar je de vraag stelt "Hey gasten, is dit icon niet beter/duidelijker etc"
--> Jij past nu een icon aan... Maar wat is de historiek? Misschien was het eerst een fa-trash en is er dan een discussie geweest en is er dan beslist om fa-minus-circle te gebruiken. Als er dan een PR gemerged wordt waar jij dit doodleuk zonder dat iemand betrokken is opnieuw aanpast dan heb je boze gebruikers, boze business, boze ProductOwner, boze analisten, boze testers die de dagen die erop volgen allemaal aan je bureau staan.

In dit geval:
De fa-trash is niet logisch voor bijvoorbeeld de ClientList -> Trash lijkt delete maar het is een soft delete (~ status wijziging).

Reply:
Inderdaad uw argumentering klopt volledig, had er zo niet over nagedacht. Bedankt om dat uit te wijzen. 

Wat wil je dat ik hiermee doe? Want zo'n verbodsbordachtig icoontje was niet altijd van toepassing in elke situatie, heb dat zelfs nog nooit gezien in een online UI om iets te verwijderen. @Laoujin